File: Baekjoon/Swift/BOJ_PS/BOJ_PS/Math/Q5063-TGN.swift
//
//  Q5063-TGN.swift
//  BOJ_PS
//
//  Created by thoonk on 2022/08/02.
//  TGN / 수학

for _ in 0..<Int(readLine()!)! {
    let input = readLine()!.split(separator: " ").map { Int(String($0))! }
    let r = input[0]
    let e = input[1]
    let c = input[2]
    
    let revenue = e - c
    
    if r < revenue {
        print("advertise")
    } else if r > revenue {
        print("do not advertise")
    } else {
        print("does not matter")
    }
}

File: Baekjoon/Python/BOJ_PS/Math/Q5063-TGN.py
# TGN / 수학

for i in range(int(input())):
    r, e, c = map(int, input().split())
    revenue = e - c

    if r < revenue:
        print("advertise")
    elif r > revenue:
        print("do not advertise")
    else:
        print("does not matter")